soci/fbs/ztoc.fbs

```diff
@@ -0,0 +1,48 @@
+namespace ztoc;
+
+table Xattr {
```

Comment:
Maybe we can use `struct` here? (https://google.github.io/flatbuffers/md__schemas.html)

Reply:
>Use this for simple objects where you are very sure no changes will ever be made

I'd stay with `table`. We need to support whatever filesystems supports and we have less control on the schema.

While extended attributes in popular OSes are all key-value pairs, apparently Haiku has types in addition to. So unless it is becoming the peformance bottleneck, I'd like to keep the flexibility still.

https://www.haiku-os.org/docs/userguide/en/attributes.html